lib/fluent/plugin/parser_logplex.rb

```diff
@@ -0,0 +1,38 @@
+module Fluent
+  module Plugin
+    class Logplex < Parser
+      HTTPS_REGEXP = /^([0-9]+)\s+\<(?<pri>[0-9]+)\>[0-9]* (?<time>[^ ]*) (?<drain_id>[^ ]*) (?<ident>[a-zA-Z0-9_\/\.\-]*) (?<pid>[a-zA-Z0-9\.]+)? *- *(?<message>.*)$/
```

Comment:
Can you provide an example of what you're trying to match here?

Reply:
Sure. This is inherited code, though.